lib/DDG/Spice/GithubIssues.pm

```diff
@@ -0,0 +1,47 @@
+package DDG::Spice::GithubIssues;
+# ABSTRACT: Search Github for Issues related to the query
+
+use DDG::Spice;
+use strict;
+use warnings;
+use Text::Trim;
+
+spice is_cached => 1;
+spice proxy_cache_valid => '200 1d';
+
+spice wrap_jsonp_callback => 0;
+
+spice to => 'https://api.github.com/search/issues?q=$1&callback={{callback}}';
+
+my @languages = share("languages.txt")->slurp;
+chomp(@languages);
+my $langs = join("|", map(quotemeta, @languages));
```

Comment:
You need to sort the languages by decreasing length to ensure longer strings are captured first. e.g. "coldfusion" vs "coldfusion cfc" -- that might be the only case though...